lib/SILOptimizer/Utils/Differentiation/Common.cpp

```diff
@@ -81,18 +81,6 @@ void forEachApplyDirectResult(
       resultCallback(result);
 }
 
-/// Returns the canonical derivative generic signature for the given witness
-/// and original function.
-/// - Return the witness derivative generic signature if it exists.
-/// - Otherwise, return the original function's generic signature.
-CanGenericSignature
-getDerivativeGenericSignature(SILDifferentiabilityWitness *witness,
```

Comment:
Note: this helper is removed because the witness derivative generic signature can be used directly, now that derivative generic signatures are propagated correctly. There's no need to fallback to the original generic signature.